pkg/program/shell/run_dir_unix.go

```diff
@@ -15,10 +15,32 @@ var (
 	errBadPermission = errors.New("bad permission")
 )
 
-// getSecureRunDir stats elvish-$uid under the default temp dir, creating it if
+// getSecureRunDir stats elvish under the XDG runtime dir, creating it if
```

Comment:
Make the comment as self-contained as possible: One cannot understand what "XDG runtime dir" exactly means without looking up the standard. Just say "the `XDG_RUNTIME_DIR` environment variable" and it's clear what it means.

See also comment below; that will also affect how to update this comment.

Reply:
I've reworded the comment in the latest push.